images/build/distroless-iptables/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - release-engineering-approvers
+  - BenTheElder
+reviewers:
+  - release-engineering-reviewers
```

Comment:
We should have sig network people for reviewer and approvers, 
there is an alias,

Reply:
sgtm +1

Reply:
added the alias in OWNERS_ALIASES and fixed here